Detections/DeviceNetworkEvents/SolarWinds_SUNBURST_Network-IOCs.yaml

```diff
@@ -29,7 +29,7 @@ query:  |
       timestamp = TimeGenerated,
       AccountCustomEntity = iff(isnotempty(InitiatingProcessAccountUpn), InitiatingProcessAccountUpn, InitiatingProcessAccountName),
       HostCustomEntity = DeviceName,
-      FileHashCustomEntity = InitiatingProcessMD5,
+      FileHashCustomEntity = InitiatingProcessMD5, HashAlgorithm = 'MD5'
```

Comment:
missing "," in the end of this line

Reply:
Happy to see that the validations caught this one 😊

Reply:
Yes...LOL